dashboard/app/views/shared/_course_wide_block.haml

```diff
@@ -18,7 +18,7 @@
 
 -elsif id == '20hour'
   .span6.miniblock
-    =link_to script_url(1) do
+    =link_to script_url('20-hour') do
```

Comment:
Hmm never mind.  Now it's working.  I don't know what was wrong before.  Updated in 28a2a12.